.github/workflows/build-release-pip-package.yml

```diff
@@ -53,65 +49,19 @@ jobs:
           pip install build
           pip install --upgrade twine
 
-    - name: Run build and deploy github release
+    - name: Run build
```

Comment:
Here it would be nice to set an env to provide the name of the version to the setup script.

One way would look something like this:

env:
   VERSION: ${{ event.release.prerelease && "snapshot" || event.release.tag_name}}